### DISCUSSION TITLE
How to change the color on navigation view?

### DISCUSSION THREAD

**Discussion by Chrlss** [lepoco/wpfui#1084] [Q&A] [answered]
How to change the color of navigationview symbol icon when clicked? Like it fills with color green when clicked
thankyou

**Comment by pomianowski** [comment, marked as answer]
Hey @Chrlss, by default WPF UI tries to use system accent colors.

https://github.com/lepoco/wpfui/blob/303f0aefcd59a142bc681415dc4360a34a15f33d/src/Wpf.Ui/Resources/Theme/Light.xaml#L515-L531

Like `SystemAccentColorPrimary` which can be set manually in App.xaml or via

https://github.com/lepoco/wpfui/blob/303f0aefcd59a142bc681415dc4360a34a15f33d/src/Wpf.Ui/Appearance/ApplicationAccentColorManager.cs#L135-L139
